src/Scheduler.ts

```diff
@@ -17,7 +21,7 @@ import { Subscription } from './Subscription';
  *
  * @class Scheduler
  */
-export class Scheduler {
+export class Scheduler implements IScheduler {
 
   public static now: () => number = Date.now ? Date.now : () => +new Date();
```

Comment:
one question for this is, if this becomes instance method will this be breaking change for next major semver?